src/amm.rs

```diff
@@ -134,8 +133,7 @@ pub fn calculate_liquidity_in(asset_a_reserve: Balance, asset_b_reserve: Balance
     to_u128!(b_required_hp)
 }
 
-// TODO what is being calculated?
-/// Calculating ?
+/// Calculating amount of assets returned when removing liquidity.
```

Comment:
@enthusiastmartin Is this too concrete? -> "when removing liquidity"

it might be if this is function is to be used in a wider context, but afaik this is the only context at the moment and the usability of this function outside of this context is probably questionable.

Reply:
It's ok. We'll probably rename this once we have more pool types to something like xyk_math and then it will be clear. This lib is only to be used in a pool scope.